examples/src/ray_scene/mod.rs

```diff
@@ -482,10 +475,14 @@ impl wgpu_example::framework::Example for Example {
                 for y in 0..side_count {
                     let instance = self
                         .tlas_package
-                        .get_mut_single((x + y * side_count) as usize)
+                        .get_mut_single((x + y) * side_count)
                         .unwrap();
```

Comment:
With this change the example should behave like before
```suggestion
                        .get_mut_single(x + y * side_count)
```